accepted/2021/net6.0-tfms/net6.0-tfms.md

```diff
@@ -0,0 +1,396 @@
+# .NET 6.0 Target Frameworks
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+In [.NET 5.0][net5.0] we defined a new syntax for target frameworks (TFM) that
+makes it unnecessary for both tools and humans to use a decoder ring, such as
+the .NET Standard version table, to figure out what is compatible with what. In
+this document, we're listing which TFMs we're adding in .NET 6.0.
+
+## Scenarios and User Experience
+
+### Using an existing library in Mac Catalyst
+
+Sophia is building an application for Mac Catalyst. From her experience with
+Xamarin iOS she knows the library `Xamarin.FFImageLoading` and wants to use it.
+The package installs successfully, but upon building she gets the following
+warning:
+
+> Package 'Xamarin.FFImageLoading' was restored using 'xamarin.ios' instead of
+> the project target framework 'net6.0-maccatalyst'. This package may not be
+> fully compatible with your project.
+
+To find out more, she visits the package's project site, which is on GitHub. She
+decides to file an issue to ask whether the library will add support for Mac
+Catalyst, but for the time being she's unblocked and can continue to develop her
+app.
+
+## Requirements
+
+### Goals
+
+* Support the definition of frameworks relevant for Xamarin development
+* Mac Catalyst is able to consume libraries built for Xamarin iOS, even though
+  the binary might not work. The developer should get a warning, but the tools
+  shouldn't block it.
+
+### Non-Goals
+
+* Re-design the TFM syntax
+
+## Stakeholders and Reviewers
+
+* MSBuild / SDK team
+* NuGet team
+* Project system team
+* Xamarin team
+
+## Design
+
+| TFM                | Compatible With                          |
+|--------------------|------------------------------------------|
+| net6.0             | (subsequent version of net5.0)           |
+| net6.0-windows     | (subsequent version of net5.0-windows)   |
+| net6.0-android     | xamarin.android                          |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-ios         | xamarin.ios                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-macos       | xamarin.mac                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-maccatalyst | xamarin.ios                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-tvos        | xamarin.tvos                             |
+|                    | (+everything else inherited from net6.0) |
+
+### Compatibility rules
+
+The [built-in compatibility rules][net5.0] of the `netX.Y-platform` TFMs make a
+TFM automatically compatible with:
+
+1. Itself
+2. Earlier versions of itself
```

Comment:
Maybe we should make it clear this list order does not correspond to precedence?

Reply:
I believe it is the default precedence for a regular `netX.Y-platform`, no?